### PR TITLE
add one way to returning message with fire time

### PR DESCRIPTION
**Pull request by gabrywu**
Sometimes we need to know the trigger time, with which we can calculate the delay of scheduling. So I submitted a pull

### CODE REVIEW

**Comment by gabrywu** [comment]
@enragedginger with the fire time ,we can known which job instance is being processed,so when an error occours,we can recover that with the same event,because the event contains the fire time.

File: src/main/scala/QuartzJob.scala [hunk 1]
```diff
@@ -81,7 +81,11 @@ class SimpleActorMessageJob extends Job {
        * JobDataMap uses AnyRef, while message can be any (though do we really want to support value classes?)
        * so this casting (and the initial save into the map) may involve boxing.
        **/
-      val msg = dataMap.get("message")
+      val msg = dataMap.get("message") match {
```
[the remark this answers is not in the document]
Comment (gabrywu) [reply src/main/scala/QuartzJob.scala]: thanks a lot.but anyref  ?  or  any?

**Comment by enragedginger** [comment]
@gabrywu Yeah, that makes sense to me. Can you update the README and add a section explaining how to use this feature so that other users of the library can benefit from it too?

**Comment by enragedginger** [comment]
Cool. Thank you!

**Comment by kvma** [comment]
@gabrywu thanks for this contribution!  @enragedginger - I'm wondering what is the timeline for cutting a new release with this change?

**Comment by polymorphic** [comment]
I'm also interested in seeing this feature in the next release.

**Pull request merged**